superset/assets/src/visualizations/leaflet_map.js

```diff
@@ -108,34 +108,34 @@ function leafletmap(slice, payload) {
         }
     }
 
+    function getRangeValue(val, max, min) {
+      if(max - min === 0) {
+        return 1;
+      }
+      return (val - min) / (max - min);
+    }
```

Comment:
We need to take care of negative overflow when val > max  and val < min

Reply:
changes done